vvgo.go

```diff
@@ -1,14 +1,73 @@
 package main
 
 import (
-	"log"
+	"github.com/sirupsen/logrus"
 	"net/http"
+	"os"
+	"strconv"
 )
 
+const Location = "us-east-1"
+
+var logger = &logrus.Logger{
+	Out: os.Stderr,
+	Formatter: &logrus.TextFormatter{
+		ForceColors:   true,
+		FullTimestamp: true,
+	},
+	Hooks:        make(logrus.LevelHooks),
+	Level:        logrus.InfoLevel,
+	ExitFunc:     os.Exit,
+	ReportCaller: false,
+}
+
+type Config struct {
+	Minio MinioConfig
+	Api   ApiServerConfig
+}
+
+func NewDefaultConfig() Config {
+	return Config{
+		Minio: MinioConfig{
+			Endpoint:        "localhost:9000",
+			AccessKeyID:     "minioadmin",
+			SecretAccessKey: "minioadmin",
+			UseSSL:          false,
```

Comment:
Are these the Minio out-of-box defaults?

Reply:
Ya. These will connect you to a new minio container,